example/1.terraform-automation/README.md

```diff
@@ -0,0 +1,95 @@
+# Terraform Automation Source
+
+
+## Introduction
+
+This repository contains the terraform modules which helps in automating the GCP resource creation. These terraform module will create
+- Cloud Composer
+- GCS bucket
+- Cloud Build Trigger
+- Cloud Source Repository
+- Service Account for Cloud Composer
+- Pub-Sub respource
+- Network resources like VPC and Subnet
+- Enable Google API's
+
+## Pre requistie
+
+1. You must have a GCP project created with `project owner` permissions.
```

Comment:
Could we suggest reader to use cloud-shell as the default option, as tool is be default installed here.